decepticonlp/transforms/perturb.py

```diff
@@ -79,20 +79,29 @@ def delete(word):
 
 
 def visual_similar_chars(word, *arg):
+    """
+    unicode_array is a list of different unicodes.
+    each char of the word is perturbed by a unicode chosen at random
+    from the unicode_array.
+    
+    eg:
+    input : adversarial
+    output : a̐d̅v̕e̒ŕŝa̅r̕îál̂
+    """
+
+    unicode_array = np.array(
+        [u"\u0301", u"\u0310", u"\u0305", u"\u0315", u"\u0312", u"\u0302"]
+    )
+
+    char_array = np.array(list(word))
+
+    int_pick = np.random.randint(0, high=unicode_array.shape[0], size=len(word))
```

Comment:
Use `np.random.choice`

Reply:
How does this make a difference? Not sure about this

Reply:
This will shorten your code, random.choice was made for this, you can implement the same code in just a few lines, also will be marginally faster

Reply:
Okay cool